WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -32,10 +33,25 @@ public static void Show(string title, string message, Action? onClick = null)
 	{
 		if (NotificationManager is { } nm)
 		{
-			RxApp.MainThreadScheduler.Schedule(() => nm.Show(new Notification(title, message, NotificationType.Information, TimeSpan.FromSeconds(DefaultNotificationTimeout), onClick)));
+			RxApp.MainThreadScheduler.Schedule(() => nm.Show(new Notification(title, message, NotificationType.Information, TimeSpan.FromSeconds(DefaultNotificationTimeout), () => TryExecute(onClick))));
 		}
 	}
 
+	private static void TryExecute(Action? action)
+	{
+		if (action is null)
+		{
+			return;
+		}
+
+		if (MainViewModel.Instance.IsBusy)
+		{
+			return;
+		}
+
+		action();
+	}
```

Comment:
I think this is not correct, at least not correct all the time.

Currently, we do navigation on click, yes, but it can change anytime and the new action might be executed even when any screen is busy. (e.g. Open the browser or anything).

The IsBusy check should be where we are sure navigation will happen.
https://github.com/zkSNACKs/WalletWasabi/blob/1693c9151b6487df29e0bb0a0c7166ef5605ceb9/WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs#L110